src/_guides/language/language-tour.md

```diff
@@ -516,7 +516,7 @@ const aConstString = 'a constant string';
 var aNum = 0;
 var aBool = true;
 var aString = 'a string';
-const aConstList = const [1, 2, 3];
+const aConstList = [1, 2, 3];
```

Comment:
Note here, too?

Reply:
I wouldn't add it here. Most folks naturally write `const aConstList = [1, 2, 3]` (forgetting the `const` on the list), only to be reminded (in Dart 1) by the IDE/analyzer about it missing.